cli_client/python/timesketch_cli_client/commands/sketch.py

```diff
@@ -53,3 +55,42 @@ def create_sketch(ctx, name, description):
         description = name
     sketch = api_client.create_sketch(name=name, description=description)
     click.echo(f"Sketch created: {sketch.name}")
+
+
+@sketch_group.command("add_event")
+@click.option("--message", required=True, help="Message of the event.")
+@click.option(
+    "--date",
+    required=True,
+    help="Date of the event (ISO 8601). Example: 2023-03-08T10:59:24+00:00",
+)
+@click.option(
+    "--attributes",
+    required=False,
+    help="Attributes of the event. Example: key1=value1,key2=value2",
+)
+@click.option(
+    "--timestamp_desc",
+    required=False,
```

Comment:
Shouldn't this be `True` ?

Reply:
well no, if one decides to not provide one, I will set it further down... but I can also enforce it 🤷